### DISCUSSION TITLE
Terraria, tshock plugins.....a little help please

### DISCUSSION THREAD

**Discussion by 1117smatt** [Pryaxis/TShock#2146] [Q&A]
I'm looking for a working version of the following plugins.  History, Diemob, Server broadcast 
Any help would be very appreciative!

**Comment by Quinci135** [comment]
https://github.com/Patrikkk/DieMob
https://github.com/Patrikkk/History
https://github.com/Patrikkk/AutoBroadcast
I'd look at the original repository of the plugins then look at the 'networks' tab to see which forks have been recently updated. Pretty sure they should work but I haven't personally tested.

**Reply by 1117smatt** [reply]
thank you so very much!
